<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsMenu</class>
 <widget class="QDialog" name="SettingsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="music_on">
      <property name="text">
       <string>On</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="music_off">
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>175</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="card_back_preview">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>20</y>
     <width>50</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bg_preview">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="card_front_preview">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QComboBox" name="card_back_combo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="bg_combo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="card_front_combo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Front Texture</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>105</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Background Texture</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Back Texture</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="about_button">
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
